### DISCUSSION TITLE
how to use SDXL turbo?

### DISCUSSION THREAD

**Discussion by Kreijstal** [leejet/stable-diffusion.cpp#112] [Q&A]
Which format of weights are required? gguf? safetensors? how do you run it? I see you recently added support for it.

**Comment by leejet** [comment]
SDXL-Turbo is currently not supported. The supported version at the moment is SD-Turbo. You can directly download safetensors from https://huggingface.co/stabilityai/sd-turbo and use it. I'm currently in the process of attempting to add support for SDXL. Once SDXL is supported, SDXL-Turbo will also be supported.

**Reply by paulocoutinhox** [reply]
Nice :) Thanks.

**Comment by RedAndr** [comment]
> You can directly download safetensors from https://huggingface.co/stabilityai/sd-turbo and use it.

I'm getting an error: [ERROR] stable-diffusion.cpp:160  - get sd version from file failed: 'models/sdxl-turbo_unet.safetensors'

**Reply by Amin456789** [reply]
i think u r downloading the wrong safetenosrs, go to the page in the description of sd cpp for sdxl turbo and get this one = sd_xl_turbo_1.0_fp16.safetensors
